common/lib/dependabot/pull_request_creator.rb

```diff
@@ -176,6 +176,7 @@ def azure_creator
         pr_name: message.pr_name,
         author_details: author_details,
         labeler: labeler,
+        reviewers: reviewers || [],
```

Comment:
This was the only thing that gave me pause... IMO we need to do one of two things:
1. decide we want to support reviewers as a top-level arg across all the clients that support it... planning to eventually also add plumbing support for it in github/gitlab/bitbucket even if they're no-ops for now.
2. Not let this leak from Azure upwards... not even sure this is possible with the current code layout. But if we're only doing it for Azure, I'd rather not add an arg globally.

At first glance I think option 1 is the better route to go, but I'm going to reach out to a teammate to get a second opinion. In the meantime, no need to modify this PR, just leave as-is for now.